mods/ts/rules/nod-support.yaml

```diff
@@ -137,7 +137,7 @@ NAFNCE:
 
 NALASR:
 	Inherits: ^Defense
-	Inherits@IDISABLE: ^DisableOnPowerDown
+	Inherits@IDISABLE: ^DisableOnLowPowerOrPowerDown
```

Comment:
...this change needs to be reverted.

Laser Turrets in original TS could be powered down, however they were *not* affected by low power (unrealistic, but probably necessary for balancing reasons).

Reply:
Done